features/routing/route.feature

```diff
@@ -199,42 +198,32 @@ Feature: Testing route
     When I run the :create client command with:
       | f | service_secure.yaml |
     Then the step should succeed
-
     Given I have a pod-for-ping in the project
-    And CA trust is added to the pod-for-ping
-    Given I obtain test data file "routing/reencrypt/route_reencrypt-reen.example.com.crt"
-    Given I obtain test data file "routing/reencrypt/route_reencrypt-reen.example.com.key"
-    Given I obtain test data file "routing/reencrypt/route_reencrypt.ca"
+    #And CA trust is added to the pod-for-ping
+    # check reencrypt route
     Given I obtain test data file "routing/reencrypt/route_reencrypt_dest.ca"
     When I run the :create_route_reencrypt client command with:
-      | name       | route-recrypt                                                                                 |
-      | hostname   | <%= rand_str(5, :dns) %>-reen.example.com                                                     |
-      | service    | service-secure                                                                                |
-      | cert       | route_reencrypt-reen.example.com.crt |
-      | key        | route_reencrypt-reen.example.com.key |
-      | cacert     | route_reencrypt.ca                   |
-      | destcacert | route_reencrypt_dest.ca              |
-      | path       | /test                                                                                         |
+      | path       | /test                   |
```

Comment:
Better to move this line/option to last, after the line `| destcacert | route_reencrypt_dest.ca |`

Reply:
Ok, I will check and do it, thanks.